integrations/github/jwt.go

```diff
@@ -101,7 +107,11 @@ func (i integration) NewClientWithAppJWTFromGitHubID(appID int64) (*github.Clien
 	}
 
 	// Initialize a client with the generated JWT injected into outbound requests.
-	return github.NewClient(&http.Client{Transport: atr}), nil
+	client := github.NewClient(&http.Client{Transport: atr})
+	if enterpriseURL := os.Getenv(enterpriseURLEnvVar); enterpriseURL != "" {
```

Comment:
tip: you could use LookupEnv, but just fyi

Reply:
ack, I prefer `Getenv` because if the result is "" I don't care why that is (i.e. env var missing or empty)